### ISSUE TITLE
Mosaic reveal: keyboard nav does not work

### ISSUE THREAD

**Issue by twhite313** [Esri/esri-eds#503]
Tab through panels: each goes gray. No text, no slide, just the gray overlay

**Comment by twhite313** [comment]
broken layout has been resolved.
